pkg/storage/memory/memory.go

```diff
@@ -777,6 +777,18 @@ func (s *MemoryBackend) ListStores(ctx context.Context, options storage.ListStor
 		stores = append(stores, t)
 	}
 
+	if len(options.IDs) > 0 {
```

Comment:
Will be good to have unit test to test this (i.e., whether the filter is working properly).  Similar for the sql storage interfaces

Reply:
This is tested:
https://github.com/openfga/openfga/pull/1950/files#diff-a26545677701a4b3a9713c2221c6e16d199561147b48f1ae839378db43e0fcd2R87-R95